Contents/Services/URL/TabloLive/ServiceCode.pys

```diff
@@ -54,29 +55,26 @@ def MediaObjectsForURL(url):
 			]
 		)
 	]
-                
+
 
 
 ####################################################################################################
 @indirect
 def PlayVideo(url):
 	Log(LOG_PREFIX + "Starting GetLiveTVStream Call")
 	channelDict = Decodeobj('channel',url)
-	myurl = "http://" + channelDict['private_ip'] +":8886"
-	cmd = "/player/watchLive"
-	parms = {"channelid": channelDict['objectID']}
-	result = TabloAPI(myurl,cmd,parms)
+	url = "http://{}:8885/guide/channels/{}/watch".format(channelDict['private_ip'], channelDict['objectID'])
+	result = JSON.ObjectFromString(str(HTTP.Request(url, method='POST')))
```

Comment:
Use REST API to retrieve the m3u8 playlist.  The previously used RPC call no longer works.